benchmarks/cases/project_linear.py

```diff
@@ -0,0 +1,27 @@
+import cartopy.io.shapereader as shpreader
+import cartopy.crs as ccrs
+import shapely.geometry as sgeom
+
+
+def time_ocean_project_pc():
```

Comment:
Maybe should [parameterize this](https://asv.readthedocs.io/en/stable/writing_benchmarks.html#parameterized-benchmarks) over all projections.

Reply:
See my comment: didn't really like the stdout when parameterised, so I didn't do this.